recipes/kdsoap/all/conanfile.py

```diff
@@ -0,0 +1,57 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class KdsoapConan(ConanFile):
+    name = "kdsoap"
+    license = (
+        "https://raw.githubusercontent.com/KDAB/KDSoap/kdsoap-{0}-release/LICENSE.txt,"
+        "https://raw.githubusercontent.com/KDAB/KDSoap/kdsoap-{0}-release/LICENSE.AGPL3-modified.txt,"
+        "https://raw.githubusercontent.com/KDAB/KDSoap/kdsoap-{0}-release/LICENSE.GPL.txt,"
+        "https://raw.githubusercontent.com/KDAB/KDSoap/kdsoap-{0}-release/LICENSE.LGPL.txt").format("master")
+    author = "Klaralvdalens Datakonsult AB (KDAB) info@kdab.com"
+    exports_sources = ["CMakeLists.txt"]
+    url = "https://github.com/KDAB/KDSoap.git"
+    description = "KD Soap is a Qt-based client-side and server-side SOAP component."
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "cmake"
+    _source_subfolder = "source_subfolder"
+
+    def requirements(self):
+        self.requires("qt/5.13.2@bincrafters/stable")
```

Comment:
unfortunately, Bincrafters and Conan Community package are not allowed. We need to have Qt in CCI first.

Reply:
Until Qt is in CCI it can take some long time. Sorry about that 😅